client/sender-app/start/start.html

```diff
@@ -1,6 +1,8 @@
 <section class="container-fluid body-content">
   <div class="controls small-center">
-    <img ng-show="!vm.connectionStatus" ng-click="vm.hasExtension && vm.onChrome && vm.connect()" src="content/ic_cast_grey_24dp.png" alt="chromecast icon" draggable="false">
+    <div ng-show="!vm.connectionStatus" ng-click="vm.hasExtension && vm.onChrome && vm.connect()" ng-init="imgsrc='content/cast_disconnected_black-512.png'" ng-mouseover="imgsrc='content/cast_connected_gold-512.png'" ng-mouseout="imgsrc='content/cast_disconnected_black-512.png'">
+      <img ng-src="{{imgsrc}}" alt="chromecast icon" draggable="false"/>
+    </div>
```

Comment:
Should avoid ng-init and instead set up imgsrc in the controller, making it available on vm. https://docs.angularjs.org/api/ng/directive/ngInit